src/cli/mod.rs

```diff
@@ -1,8 +1,6 @@
 mod command;
 mod error;
 mod fmt;
-#[cfg(feature = "js")]
-pub mod javascript;
```

Comment:
We should keep this feature flag. CLI need to have a quickjs based implementation of the worker.